server/src/main/java/org/opensearch/index/compositeindex/datacube/startree/data/DataType.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.index.compositeindex.datacube.startree.data;
+
+/**
+ * Data type of doc values
+ * @opensearch.internal
+ */
+public enum DataType {
+    INT(Integer.BYTES, true),
```

Comment:
Lets remove boolean for now , as its not used right now

Reply:
Removed DataType. Using StarTreeNumericType instead. We can extend that enum if we would need support for boolean (numeric / bytes etc.)